components/images.css

```
@@ -0,0 +1,14 @@
+.image-container {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 1rem;
+}
+
+.responsive-image {
+  width: 100%;
+  object-fit: cover;
+}
+
+.responsive-round-image {
+  border-radius: 50%;
+}
```

as I saw on the preview image doesn't look like a round shape It looks like elliptical. for getting an exact round(circle) you can use fix width and height like below
```suggestion
.responsive-round-image {
width: 20rem;
height: 20rem;
  border-radius: 50%;
}
```

---

Also, you can use aspect-ratio:1/1 for making an image square.

---

Thanks  I have implemented these changes